runtime/etc/opa/opa_conf.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  - name: "${OPA_SERVICE_NAME}"
+    url: "${OPA_SERVICE_URL}"
+    credentials:
+      s3_signing:
+        metadata_credentials:
+          iam_role: "app-${APPLICATION_ID}"
```

Comment:
This role might not always exist, because sometimes users create their own roles in the senza manifest.

Reply:
On second thoughts, the instance would use the credentials from the metadata endpoint and it would have an existing role associated with it. As long as that role has permissions to read from the s3 bucket this line can be dropped.

Reply:
opa documentation specify to provide the iam_role and region: https://www.openpolicyagent.org/docs/latest/configuration/#aws-signature
`To use the EC2 metadata service, the IAM role to use and the AWS region for the resource must both be specified as iam_role and aws_region respectively.`

Reply:
Well then my first point holds. This is not the role which will always be used/present.

Reply:
Agree but then to use opa, the intended applications would have to have a role like this...